prov/bgq/include/rdma/bgq/fi_bgq_flight_recorder.h

```diff
@@ -0,0 +1,128 @@
+
+
+#ifndef PROV_BGQ_INCLUDE_FLIGHT_RECORDER_H
+#define PROV_BGQ_INCLUDE_FLIGHT_RECORDER_H
+
+//#define FLIGHT_RECORDER_ENABLE
```

Comment:
Please don't include commented-out code.

In this case, it would be better to always define `FLIGHT_RECORDER_ENABLE` to be 0 or 1 (vs. defined vs. undefined).  Then use `#if` instead of `#ifdef`.